src/main/java/pl/piekoszek/gorskimatches/image/ImageControler.java

```diff
@@ -0,0 +1,20 @@
+package pl.piekoszek.gorskimatches.image;
+
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+import java.io.IOException;
+
+@RestController
+@RequestMapping("api")
+class ImageController {
```

Comment:
File is named `ImageControler` class is named `ImageController` (double l)
Both should be `ImageController`